src/api/api.tsx

```diff
@@ -5,68 +5,154 @@ import {
   NotFound,
   PermissionDenied,
 } from '../errors/errors';
-
-const LOCALSTORAGE_TOKEN_NAME = 'zaaktypenbeheer_token';
-localStorage.setItem(LOCALSTORAGE_TOKEN_NAME, import.meta.env.VITE_API_LOGIN_TOKEN);
-
-interface APIResponse {
-  detail: string;
-  [key: string]: any;
-}
+import { CSRFToken } from './headers';
 
 class Api {
-  api_token: string | null;
   api_url: string;
+  fetchDefaults: any;
 
   constructor() {
-    this.api_token = localStorage.getItem(LOCALSTORAGE_TOKEN_NAME);
-    this.api_url = import.meta.env.VITE_BASE_API_URL;
+    this.api_url = import.meta.env.VITE_BASE_API_URL as string;
+    this.fetchDefaults = {
+      credentials: 'include', // required for Firefox 60, which is used in werkplekken
+    };
+  }
+
+  getEndpoint(url: string) {
+    if (!this.api_url.endsWith('/')) this.api_url += '/';
+    if (url.startsWith('/')) url = url.substring(1);
+    if (!url.endsWith('/') && url !== '') url += '/';
+    return this.api_url + url;
   }
```

Comment:
I think that constructing the URL could maybe be left to who is making a call. It gives a bit more flexibility :thinking: